draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1831,7 +1831,9 @@ might block third-party cookies entirely by refusing to send Cookie header field
 process Set-Cookie header fields during third-party requests. They might take a less
 draconian approach by partitioning cookies based on the first-party context,
 sending one set of cookies to a given third party in one first-party context,
-and another to the same third party in another.
+and another to the same third party in another. Or they might even send or set
```

Comment:
nit: There are a couple usages of "third party" in this paragraph, can you change them to "third-party"?

Reply:
I think "third-party" is correct when used as an adjective, but "third party" is correct when used as a noun (which the usages you're referring to seem to be).

See https://www.merriam-webster.com/dictionary/third-party, https://www.depo.com/wp-content/uploads/2015/02/hyphenate.pdf for example.

Reply:
Ah that makes sense, thanks Chris.

Consider this issue closed.

Reply:
![dog](https://user-images.githubusercontent.com/67283/133153826-1a7374f4-656c-4f2b-9975-fdfb92e48c0c.jpeg)